README.md

```diff
@@ -270,6 +270,33 @@ func(MyModel.objects.annotate(foo=Value("")).get(id=1))  # OK
 func(MyModel.objects.annotate(bar=Value("")).get(id=1))  # Error
 ```
 
+### How do I check if something is an instance of QuerySet in runtime?
+
+A limitation of making `QuerySet` generic is that you can not use
+it for `isinstance` checks.
+
+```python
+from django.db.models.query import QuerySet
+
+def foo(obj: object) -> None:
+    isinstance(obj, QuerySet) # Error: Parameterized generics cannot be used with class or instance checks
+```
+
+To get around with this issue without making `QuerySet` non-generic,
+Django-stubs provides `django_stubs_ext.QuerySetAlias`, a non-generic
+variant of `QuerySet` dedicated for runtime type checking:
+
+```python
+from django_stubs_ext import QuerySetAlias
```

Comment:
Do others have any name ideas for this?

I fear that `Alias` is too generic. It does not show the purpose of this thing.

Maybe `QuerySetInstance`?

Reply:
`QuerySetInstance` 👍

Reply:
Alternatively `QuerySetAny` as in "QuerySet of any model"?

Reply:
`QuerySetAny` sounds better, since this type is essentially `_QuerySet[Any, Any]`.